src/Goerli-DssSpell.t.sol

```diff
@@ -106,13 +107,13 @@ contract DssSpellTest is GoerliDssSpellTestBase {
     }
 
     function test_nextCastTime() public {
-        hevm.warp(1606161600); // Nov 23, 20 UTC (could be cast Nov 26)
+        hevm.warp(1638302400); // Nov 30, 20 UTC (could be cast Dec 2)
```

Comment:
Did this test need to be changed?

Reply:
I am not sure. I thought this had to match the date the spell was suppoed to be running. I simply added 1 week to the previous one.

Should I rollback this change?